Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:9
```

Comment:
Weird i don't see node 9 in the official tags https://hub.docker.com/_/node/

Reply:
Neither do I, and 8 is the most recent LTS, so I've based everything off that.